packages/ui/src/description-list/DescriptionRawItem.tsx

```diff
@@ -0,0 +1,45 @@
+import { Typography } from '@material-ui/core';
+import { forwardRef } from 'react';
+import styled from 'styled-components';
+import { Column } from '../columns/Column';
+import { Columns } from '../columns/Columns';
+import { Color } from '../theme/Color';
+
+export const DottedLine = styled.div`
+  border-bottom: 1px dashed ${Color.Silver400};
+  margin: 0px 8px;
+  height: 7px;
+`;
+
+interface DescriptionRawItemProps {
+  title: string;
+  e2eValue?: string;
+  value: string | number | JSX.Element;
+  boldValue?: boolean;
```

Comment:
Why do we need it?

Reply:
for bold type value on rawItem, on right side value could be `h4` or `body2`, `boldValue` will check on truthy and falsy

Reply:
<img width="332" alt="image" src="https://github.com/superdispatch/web-ui/assets/118124821/6a6b696e-9f9a-4932-92bc-ef55e67c32df">